flybywire-aircraft-a320-neo/html_ui/Pages/A32NX_Utils/NXSystemMessages.js

```diff
@@ -61,6 +61,7 @@ const NXSystemMessages = {
     cancelAtisUpdate:       new TypeIMessage("CANCEL UPDATE BEFORE"),
     checkMinDestFob:        new TypeIIMessage("CHECK MIN DEST FOB"),
     checkToData:            new TypeIIMessage("CHECK TAKE OFF DATA", true),
+    checkWeight:            new TypeIIMessage("CHECK WEIGHT", true),
```

Comment:
This message relates to the secondary flight plan. Are you sure it's the one you want?